docs/recipes/WritingTests.md

```diff
@@ -304,7 +304,7 @@ import Header from '../../components/Header'
 
 function setup() {
   const props = {
-    addTodo: expect.createSpy()
+    addTodo: jest.fn(() => {})
```

Comment:
Yeah I give this issue a quick try too and noticed this kind of errors (my branch is roughly the same than yours). Didn't had much time to dig further though.